pom.xml

```diff
@@ -16,7 +16,7 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<webappDirectory>${project.build.directory}/${project.build.finalName}</webappDirectory>
-		<synapse.version>366.0</synapse.version>
+		<synapse.version>375.0</synapse.version>
```

Comment:
Synapse Java client is still being used servlet side for one of these calls, so we must update to see the backend API change.